docs/experience/resources.md

```diff
@@ -0,0 +1,70 @@
+title: Resources Timing
+description: How Sematext Experience collects external resources data such as downloaded images, CSS files, and Javascript files
+-----
+
+Experience collects resources data such as downloaded images, CSS and Javascript files. When someone visits your website the browser will begin downloading resources. These resources can have a big impact on your website loading speed.
+
+The same process happens in Single Page Apps when user navigates to a different part of the web app.
+
+There are two ways to see resources performance data in Experience.
+
+
+## Resources Waterfall
+
+Open the details page for some page load to access the Resources Waterfall table.
```

Comment:
Open the details page for some page load to access the Resources Waterfall table. => Open the page-load details page to access the Resource Waterfall table.